addons/web/static/src/views/fields/relational_utils.js

```diff
@@ -801,6 +808,24 @@ export function useX2ManyCrud(getList, isMany2Many) {
         };
     }
 
+    const openFullRecord = async (record, fields) => {
```

Comment:
I don't think it's the job of "useX2ManyCrud" to provide a "openFullRecord" function.

Opening a record is not really a CRUD operation.

Can't this function be inlined where it's used instead? Or at least moved to "useOpenX2ManyRecord" where it would make sense along with "openRecord"?